test/dlc_tests/conftest.py

```diff
@@ -289,7 +289,8 @@ def ec2_instance(
                 if ec2_instance_ami == AML2_GPU_DLAMI_US_WEST_2
                 else UBUNTU_18_BASE_DLAMI_US_EAST_1
             )
-
+    
+    ec2_key_name = f"{ec2_key_name}-{str(uuid.uuid4())}"
```

Comment:
There is no import statement for `uuid`, which has caused errors in the EC2 test jobs.

Reply:
yeah, i also noticed this, maybe it was removed due to merge.